.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         env:
           - {ROS_DISTRO: foxy, ROS_REPO: main}
           - {ROS_DISTRO: foxy, ROS_REPO: testing}
+          - {ROS_DISTRO: foxy, ROS_REPO: main}
```

Comment:
this was already there no?

Reply:
is that a GH displaying thing? I can't seem to find the change.